server/frontend/dist/shotgrid-addon.js

```diff
@@ -160,12 +296,19 @@ const getShotgridProjects = async () => {
 
   if (sgProjects) {
     sgProjects.forEach((project) => {
-      sgProjectsConformed.push({
-      "name": project.attributes.name,
-      "code": project.attributes[`${addonSettings.shotgrid_project_code_field}`],
-      "shotgridId": project.id,
-      "ayonId": project.attributes.sg_ayon_id,
-    })
+      /* Only add projects that don't contain whitespaces in the name
+      and have a code name as those are the requirements to sync to Ayon. */
+      if (
+        !project.attributes.name.includes(" ") &&
+        project.attributes[`${addonSettings.shotgrid_project_code_field}`]
+      ) {
+        sgProjectsConformed.push({
+          "name": project.attributes.name,
+          "code": project.attributes[`${addonSettings.shotgrid_project_code_field}`],
+          "shotgridId": project.id,
+          "ayonId": project.attributes.sg_ayon_id,
+        })
+      }
```

Comment:
This should not be part of this PR please.

Reply:
sure

Reply:
done